.github/workflows/canary.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Embed version
         run: |
-          echo " cachedversion = \"Canary ${GITHUB_SHA:0:7}\"" >> lua/wire/server/wirelib.lua
+          echo -e "\ncachedversion = \"Workshop ${GITHUB_SHA:0:7}\"" >> lua/wire/server/wirelib.lua
```

Comment:
Did you intend to revert the Canary to Workshop? I thought it'd be good to differentiate between the two but I don't care too much

Reply:
Just a copying error. You should have write permission so if you see an issue feel free to fix it.

Reply:
Well wasn't sure if it was an issue. Could've been a change of heart